src/Uno.Toolkit.UI/Behaviors/InputExtensions.cs

```diff
@@ -0,0 +1,150 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Input;
+using Windows.System;
+using Windows.UI.ViewManagement;
+
+#if IS_WINUI
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Input;
+#else
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Input;
+#endif
+
+namespace Uno.Toolkit.UI
+{
+	public static class InputExtensions
+	{
+		#region DependencyProperty: AutoDismiss
+
+		public static DependencyProperty AutoDismissProperty { get; } = DependencyProperty.RegisterAttached(
+			"AutoDismiss",
+			typeof(bool),
+			typeof(InputExtensions),
+			new PropertyMetadata(default(bool), OnAutoDismissChanged));
+
+		public static bool GetAutoDismiss(DependencyObject obj) => (bool)obj.GetValue(AutoDismissProperty);
+		public static void SetAutoDismiss(DependencyObject obj, bool value) => obj.SetValue(AutoDismissProperty, value);
+
+		#endregion
+		#region DependencyProperty: AutoFocusNext
+
+		public static DependencyProperty AutoFocusNextProperty { get; } = DependencyProperty.RegisterAttached(
+			"AutoFocusNext",
+			typeof(bool),
+			typeof(InputExtensions),
+			new PropertyMetadata(default(bool), OnAutoFocusNextChanged));
+
+		public static bool GetAutoFocusNext(DependencyObject obj) => (bool)obj.GetValue(AutoFocusNextProperty);
+		public static void SetAutoFocusNext(DependencyObject obj, bool value) => obj.SetValue(AutoFocusNextProperty, value);
```

Comment:
Please follow the standard by prefixing you property with `Is` (i.e. `IsAutoFocusNextEanbled`). 

BTW do we really need such property, compared to just relying on the fact that the `IsAutoFocusElement` is set ?
And if you do want to have 2 properties, the default value of this should be `true`.

Reply:
(The "Is" prefix will be dsicussed there: https://github.com/unoplatform/uno.toolkit.ui/pull/287#discussion_r950437134)

tl;dr: 
```cs
if (AutoFocusNext || AutoFocusNextElement != null)
     Focus(AutoFocusNextElement ?? FindNextFocusable());
```

For having two property here, they are needed because we have 2 DIFFERENT behaviors:
AutoFocusNext and AutoFocusNextElement have different focus target:
- AutoFocusNext is determined by `FocusManager.FindNextFocusableElement`
- AutoFocusNextElement is provided by the value.